<article class="bugdoc">
    <h2 class="bugdoc-title" id="Bug-Doc:PE22316">PE22316: B001</h2>
    <h3>B001</h3>
    <section id="tip">
     <h4 style="border-top: 0pt; padding: 0pt 4pt 4pt 4pt;">Bug info</h4>
                <dl>
                    <dt>BugTypeID</dt>
                    <dd>PE22316</dd>
                    <dt>Tags</dt>
                    <dd>flake8</dd>
                    <dt>Category</dt>
                    <dd>Code style and efficiency</dd>
                    <dt>Authors</dt>
                    <dd>pinpoint</dd>
                </dl>
    <h2>Reason</h2>

<pre><code>B001 Do not use bare `except:`, it also catches unexpected events like memory errors, interrupts, system exit, and so on.Prefer `except Exception:`.If you're sure what you're doing, be explicit and write `except BaseException:`.

</code></pre>
<h4>References</h4>
<section class="footnotes">
<ol>
<li id="fn-1"><p>https://www.python.org/dev/peps/pep-0008/<a href="#fnref-1" class="footnote">&#8617;</a></p></li>
</ol>
</section>

    </section>
</article>
